Makefile

```diff
@@ -7,7 +7,7 @@ BUILD_DIR=./build/api
 .ALL: help
 
 # Versions that can be built.
-VERSIONS = 1.2 1.3 2.0 2.1 2.2 2.3 2.4 2.5 2.6 2.7 3.0
+VERSIONS = 1.2 1.3 2.0 2.1 2.2 2.3 2.4 2.5 2.6 2.7 3.0 3.1
 
 # Versions that are actively developed / maintained.
 ACTIVE_VERSIONS = 2.7 3.0
```

Comment:
I think 3.1 should also be added to `ACTIVE_VERSIONS` list as it's actively developed.